package.json

```diff
@@ -29,6 +29,7 @@
     "react-dom": "^16.6.3"
   },
   "devDependencies": {
+    "@zeit/next-css": "^1.0.1",
```

Comment:
Is this a dev dependency?

Reply:
@shyaman No I tried to remove, I haven't used it actually, just added during testing to add a feature. When I tried to remove it, there was a travis build failure. Check if it is possible to remove it without build failure

Reply:
@shyaman Seems like issue is resolved, removed the unwanted @zeit/next-css package without any build failure. Please check and resolve if problem still exists.